.github/workflows/pull_request.yml

```diff
@@ -41,7 +41,7 @@ jobs:
           AUTH0_USERNAME: ${{ secrets.AUTH0_USERNAME }}
         run: |
           MINIKUBE_PG_URL="${{ secrets.TEST_DB_URL }}"
-          MINIKUBE_API_HOST="https://staging.meroxa.io"
+          MINIKUBE_API_HOST="${{ secrets.API_HOST }}"
```

Comment:
this should be `https://api.staging.meroxa.io` not secrets

Reply:
@terrancej I had tried that! but here's my another attempt https://github.com/meroxa/cli/pull/81/commits/a84a94afb6751887554d86879a9cafdf7a54201e